<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="consciousLabel">
       <property name="text">
        <string>Check Consciousness</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emergencyLabel">
       <property name="text">
        <string>Call Emergency</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="airwayLabel">
       <property name="text">
        <string>Open Airway</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="breathLabel">
       <property name="text">
        <string>Check Breath</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="padLabel">
       <property name="text">
        <string>Pad Placement</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="heartAnalysisLabel">
       <property name="text">
        <string>Heart Analysis</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="ShockButton">
       <property name="text">
        <string>Shock</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="cprLabel">
       <property name="text">
        <string>CPR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>376</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="ConsCheck">
         <property name="text">
          <string>Conscious</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="BreathCheck">
         <property name="text">
          <string>Normal Breath</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="PadBox">
         <item>
          <property name="text">
           <string>Just Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Too Low</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Too High</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="HeartRhythm">
         <item>
          <property name="text">
           <string>Ventricular Fibrillation </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ventricular Tachycardia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Asystole (Flat-lined)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pulseless Electrical Activity</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ContactCheck">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CompressionCheck">
         <item>
          <property name="text">
           <string>Just Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Too Slow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Too Fast</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Conscious Check</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Breath Check</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Pad Placement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Heart Rhythm Detected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>First-aider in contact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Compression Correction</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>361</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>FIRST AIDER POINT OF VIEW</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Console Log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ConsoleLogs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Picture">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>AED INTERFACE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="batteryIcon">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLevel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>FIRST AIDER ACTIONS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Check Consciousness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Call Emergency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Open Airway</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Checking Breath</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Place Pad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Remove Pad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Change Battery</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="ConsButton">
       <property name="text">
        <string>Conscious</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EmergencyButton">
       <property name="text">
        <string>Call Emergency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AirButton">
       <property name="text">
        <string>Airway</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BreathButton">
       <property name="text">
        <string>Checking Breath</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Place_Pad">
       <property name="text">
        <string>Place Pad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Remove_Pad">
       <property name="text">
        <string>Remove Pad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeBattery">
       <property name="text">
        <string>Change Battery</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>421</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>550</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Button</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>20</width>
      <height>611</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>351</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Display"/>
   </widget>
   <zorder>label_8</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>formLayoutWidget_2</zorder>
   <zorder>label_17</zorder>
   <zorder>label</zorder>
   <zorder>ConsoleLogs</zorder>
   <zorder>Picture</zorder>
   <zorder>label_20</zorder>
   <zorder>batteryIcon</zorder>
   <zorder>batteryLevel</zorder>
   <zorder>line</zorder>
   <zorder>label_21</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>powerButton</zorder>
   <zorder>line_2</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
